### PR TITLE
Extend check if ball is a Basketball

### PR DESCRIPTION
**Pull request by Destiphy**

### CODE REVIEW

**Comment by Destiphy** [comment]
The replay of https://github.com/SaltieRL/carball/issues/38 provides this typename:

image

File: carball/json_parser/game.py [hunk 1]
```diff
@@ -403,7 +403,8 @@ def parse_replay(self):
                         # ball_is_sleeping = RBState.get('Sleeping', True)
                         data_dict = BallActor.get_data_dict(actor_data, version=self.replay_version)
                         player_ball_data['ball'][frame_number] = data_dict
-                    elif actor_data["TypeName"] == "Archetypes.Ball.Ball_Basketball":
+                    elif actor_data["TypeName"] == "Archetypes.Ball.Ball_Basketball" or \
```
Comment (dtracers) [inline carball/json_parser/game.py]: alternative is that you could do .lower and then compare to the type .lower
So it is 100% case insensitive

Comment (Destiphy) [reply carball/json_parser/game.py]: but shouldn't we do it at every type then?

Comment (dtracers) [reply carball/json_parser/game.py]: Yes most likely

**Pull request merged**